templates/tsConstructs/methodSignature.ejs

```diff
@@ -1,21 +1,16 @@
 <section class="code-doc ">
     <a name="<%-anchorId%>"></a>
 <%
-if(signatures && signatures[0]){
-    var type = signatures[0].type;
+var type = signature.type;
 %>
-    <h4 class="code-ref"><%-name%>(<%-tsHelpers.commaSepParams(signatures[0].parameters)%>) : <%-tsHelpers.returnTypeStr(type)%></h4>
+<h4 class="code-ref"><%-name%>(<%-tsHelpers.commaSepParams(signature.parameters)%>) : <%-tsHelpers.returnTypeStr(type)%></h4>
 <%
-}
+var comment = comment || signature.comment || null;
```

Comment:
Why are we doing `comment = comment || signature.comment || null` vs `comment = signature.comment || null` here?